.gitignore

```diff
@@ -3,4 +3,4 @@
 npm-debug.log
 .DS_Store
 rethinkdb_data
-.env
```

Comment:
let's keep .env in .gitignore since it's often used for local env regardless of project.